app/src/main/java/com/github/siela1915/bootcamp/AutocompleteApi/BooleanWrapper.java

```diff
@@ -0,0 +1,17 @@
+package com.github.siela1915.bootcamp.AutocompleteApi;
+
+public class BooleanWrapper {
+    private boolean bool;
+
+    public BooleanWrapper(boolean bool) {
+        this.bool = bool;
+    }
+
+    public boolean getBool() {
+        return bool;
+    }
+
+    public void setBool(boolean bool) {
+        this.bool = bool;
+    }
+}
```

Comment:
Why create a wrapper? Why not use Boolean class straight away? No need to call any function to get the Boolean value as the Java compiler does this implicitly for you.

Reply:
This was causing issues since the boolean could not be rewritten in a child method (the on item click listener).